cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/servlet/response/ResponseUtils.java

```diff
@@ -50,16 +61,25 @@ static String getBaseJSONString(String message) {
     return new Gson().toJson(jsonResponse);
   }
 
-  private static void writeResponseToOutputStream(HttpServletResponse response, int responseCode, boolean json, String responseMsg)
+  private static void writeResponseToOutputStream(HttpServletResponse response,
```

Comment:
Wouldn't it suffice to pass `String httpCorsExposeHeaders` here (please see the above proposal)?

Reply:
reason is mentioned in above comment.